pkg/api/playlist_play.go

```diff
@@ -66,6 +66,7 @@ func (hs *HTTPServer) populateDashboardsByTag(ctx context.Context, orgID int64,
 	return result
 }
 
+// Deprecated -- the frontend can do this better
 func (hs *HTTPServer) LoadPlaylistDashboards(ctx context.Context, orgID int64, signedInUser *user.SignedInUser, playlistUID string) (dtos.PlaylistDashboardsSlice, error) {
 	playlistItems, _ := hs.LoadPlaylistItems(ctx, playlistUID, orgID)
```

Comment:
Not sure how we approach deprecations on the API, but IMO we should also add deprecation notice to  `/playlists/{uid}/dashboards` route if it's no longer a preferred way, WDYT?